src/resources/postcss/views/_month.pcss

```diff
@@ -4,4 +4,62 @@
  * @package TribeEventsCalendar
  *
  */
-.tribe-events {}
\ No newline at end of file
+.tribe-events {
+
+	.tribe-events-calendar-month__calendar-event {
+		margin: var(--spacer-1) 0;
+		position: relative;
+	}
+
+	.tribe-events-calendar-month__calendar-event-featured-image-wrapper {
+		margin-bottom: var(--spacer-0);
+	}
+
+	.tribe-events-calendar-month__calendar-event-featured-image {
+		padding-bottom: 56.25%;
+	}
+
+	.tribe-events-calendar-month__calendar-event-datetime {
+		@mixin mobile-body-3;
+
+		color: var(--color-text-secondary);
+
+		> * {
+			vertical-align: middle;
+		}
+	}
+
+	.tribe-events-calendar-month__calendar-event-datetime-featured,
+	.tribe-events-calendar-month__calendar-event-datetime-recurring {
+		display: inline-block;
+		margin-left: 3px;
+	}
+
+	.tribe-events-calendar-month__calendar-event-title {
+		font-weight: var(--font-weight-regular);
+	}
+
+	.tribe-events-calendar-month__calendar-event-tooltip {
+		background-color: var(--color-background);
+		border: 1px solid var(--color-border-default);
+		box-shadow: var(--box-shadow-tooltip);
+		display: none;
```

Comment:
If we are going with a simple CSS implementation for the tooltip, for simple a11y, I'd recommend one of two things:

1) Don't do display none, instead go with the height 0/opacity 0 approach where the content is still accessible to say a screen reader

OR

2) Keep this approach and add a `aria-hidden="true"` to the tooltip container. Technically we'll still link off to the event itself where a user can get all that event's info, so I think it would be fine to not worry about making that particular functionality and content be more functionally accessible.

I'd rec the 2nd approach ; )

Reply:
We're tied to what tooltipster brings here. I'm not sure if it'll work if we do something different from `display: none`. I'm implementing this and will go with the 2nd approach

Reply:
FYI: Tooltipster is creating a `div` _on-the-fly_ outside the wrapper (at a body level), so I'll need to take this as a special case for some classes.

Reply:
Oh ok, didn't' realize that, there is this fwiw: https://iamceege.github.io/tooltipster/#accessibility

Reply:
hey @juanfra do you think you could handle this? I can merge this and you can handle the CSS as you need for the tooltip.

Reply:
Sure thing, the wrapper will be some sort of "tooltipster theme", so probably that will live outside, but I can make the rest play nicely with your work. Thanks @paulmskim !